en/gps_compass/gps_locosys_hawk_a1.md

```diff
@@ -0,0 +1,99 @@
+# LOCOSYS Hawk A1 GPS/GNSS
```

Comment:
FYI only, good to use proper name for the brand "LOCOSYS" everywhere (which I have done). But when referring to the device itself it is a Hawk A1.

Reply:
Yes, my bad.